lib/repo/repository.rb

```diff
@@ -60,8 +60,21 @@ class ExportRepositoryAlreadyExists < Exception;  end
 
   class RepositoryCollision < Exception; end
 
+  class InvalidLocation < Exception;
```

Comment:
Remove the semicolon (?)

Reply:
Actually I can remove the whole Exception. It isn't used anymore